packages/client/src/pages/olo/OloLocationInput.test.tsx

```diff
@@ -66,7 +63,7 @@ describe("OloLocationInput", () => {
     expect(inputHouseNumber).toBeInTheDocument();
     expect(inputHouseNumber).not.toHaveValue();
 
-    expect(matomoTrackEvent).toHaveBeenCalledTimes(0);
+    // expect(matomoTrackEvent).toHaveBeenCalledTimes(0);
```

Comment:
You can remove those matomo event tests

Reply:
I'd like to mock Matomo and restore these tests.